src/Client/ClientBase.h

```diff
@@ -6,13 +6,13 @@
 #include <Common/ProgressIndication.h>
 #include <Common/InterruptListener.h>
 #include <Common/ShellCommand.h>
+#include <Common/QueryFuzzer.h>
```

Comment:
I wonder if `Common` is a proper place to move this file.
Wdyt about moving `QueryFuzzer` to `Storages` (the client already depends on Storages)?

Reply:
> Wdyt about moving QueryFuzzer to Storages

Because `QueryFuzzer` by itself is not a storage (and doesn't relate to storages in any way). 
I believe that `Common` is the best place for such utilities.